roles/DCOS.agent_windows/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+- name: Windows | Defining DC/OS by using 'version' or, more specifically, 'image_commit'
+  set_fact:
+     dcos_version_specifier: "{{ dcos['version'] }}"
+  no_log: True
+
+- name: Windows | Install vcredist140
+  win_chocolatey:
+    name: vcredist140
+    state: present
+
+- name: Windows | Create folders and subfolder for DCOS binaries
+  win_file:
+    path: C:\dcos\{{ item }}
+    state: directory
+  with_items:
+    - work
+    - images
+    - mesos-logs
+
+- name: Windows | Check mesos-binaries exist
+  win_stat:
+    path: C:\dcos\mesos-binaries.{{ dcos_version_specifier }}.zip
+  register:
+    zip_file_stat
+
+- name: Windows | Download mesos-binaries
+  win_get_url:
```

Comment:
yeah, I see it will work . But it's a bit incorrect to set force =Yes by default , IMHO

Reply:
you did not read it correct ;)
the default is `force = Yes | True | true` already. You need to set it to `No` and then we can get rid of the extra action.

Reply:
I read it correctly . What I was trying to say is - `win_get_url` is designed incorrectly, because `force` flag set to yes by default. It should be set to no by default instead

Reply:
however, tested and fixed as per your recommendation

Reply:
ah okay :) so I read your comment wrong ^^ sry